src/modules/validate/validate.controller.ts

```diff
@@ -23,25 +24,30 @@ export class ValidateController {
    * @Output: password hashed
    */
   async getHashPassword(@Query() getHashPasswordDto: GetHashPasswordDto): Promise<IResponseToClient> {
+    const data = await this.validateService.hashPassword(getHashPasswordDto.password);
     return {
-      message: '',
-      data: 'ok',
+      message: ValidateSuccessMessage.HashPasswordSuccessfully,
+      data,
       statusCode: HttpStatus.OK,
     }
   }
 
   @Get("verify-hash-password")
   @Public()
+  @ApiOperation({
+    summary: 'Api to compare hash password & hashed password'
+  })
   /**
    * @Require: Using bcrypt
    * @Query: verifyPassword: string
    * @Query: passwordHashed: string
    * @Output: bool (true if password match, false if not)
    */
-  async verifyHashPassword(@Body() minusDto: MinusDto): Promise<IResponseToClient> {
+  async verifyHashPassword(@Query() verifyHashPasswordDto: VerifyHashPasswordDto): Promise<IResponseToClient> {
+    const data = await  this.validateService.comparePasswordHashed(verifyHashPasswordDto.verifyPassword, verifyHashPasswordDto.passwordHashed);
     return {
-      message: '',
-      data: 'ok',
+      message: ValidateSuccessMessage.CompleteCompare,
```

Comment:
Message display for user to see should explain result for user is success or not. We shouldnt let frontend side handle this case base on data return true or false. Message display for user should come from backend side, the reason is easy to update, message located in single resource and also for multiple language dynamic in the future. By the way, what I want is the message return back to client should explain the result is success or false, if false we can return HttpException with message password not match and so on.